packages/admin-app/src/App.tsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { DefaultLayout } from './layouts';
+import { ThemeProvider } from '@material-ui/core';
+import { AuthenticatedLayout } from './layouts';
+import { LoginPage } from './pages';
+import useToken from './useToken';
 
-import MainRoutes from './routes/MainRoutes';
+import AuthenticatedRoutes from './routes/AuthenticatedRoutes';
+import UnauthenticatedLayout from './layouts/UnauthenticatedLayout';
+import theme from './theme';
 
-const App: React.FunctionComponent = () => (
-    <Router>
-        <DefaultLayout>
-            <MainRoutes />
-        </DefaultLayout>
-    </Router>
-);
+const App: React.FunctionComponent = () => {
+    const { setToken, token } = useToken();
+
+    const getContent = () => {
+        const unauthenticated = !token;
+        if (unauthenticated) {
+            return (
+                <UnauthenticatedLayout>
```

Comment:
@keithrfung said there may be additional unauthenticated pages and so to use two layouts: authenticated and unauthenticated.  We may eventually add a Router and UnauthenticatedRouts, but I chose to defer that until we need it.